management/server/http/api/openapi.yml

```diff
@@ -110,6 +110,12 @@ components:
             ssh_enabled:
               description: Indicates whether SSH server is enabled on this peer
               type: boolean
+            user_id:
```

Comment:
instead of user_id, we can use the activated_by which accounts for setup key or user